packages/ts-moose-lib/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@514labs/moose-lib",
-  "version": "0.0.2",
+  "version": "0.0.2-rc",
```

Comment:
Do we need to update the release script to handle these tags?

Reply:
Nope that was just for testing - I can change that back